### ISSUE TITLE
In a standalone Blazor WebAssembly app IIS Express doesn't restart if the process is killed

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#17546]
Repro steps:

- Create a standalone Blazor WebAssembly app in Visual Studio
- Run it (Ctrl-F5)
- Kill the IIS Express process
- Run the app again

Expected result:

- IIS Express is restarted and the app runs normally

Actual result:

- IIS Express is not restarted. The browser is launched and pointed at an endpoint that does not exist.

**Comment by SteveSandersonMS** [comment]
I totally agree this is a bug and should be fixed.

However, more info: this has been the case forever and isn't specific to Blazor WebAssembly AFAIK. VS *will* restart IISExpress, but only after some time period has elapsed. I think it's about 15-30 seconds. If you try Ctrl+F5 again after that much time has passed, it works.

**Comment by mkArtakMSFT** [comment]
This has been an issue forever and it's not a priority as it works after several seconds passed. We'll try to follow up on this when we find time.

**Comment by mkArtakMSFT** [comment]
@danroth27 do you want to follow up on this with whoever owns it?
/cc @vijayrkn
